src/MacroTools/Powers/Transfiguration.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using Microsoft.VisualBasic;
+using WCSharp.Events;
+using WCSharp.Shared.Data;
+using static War3Api.Common;
+
+namespace MacroTools.Powers
+{
+  /// <summary>
+  /// Causes dying units to Transfiguring at a specific location.
```

Comment:
```suggestion
  /// Causes dying units to spawn a new unit at a predefined location.
```
Transfigure doesn't mean anything to the reader.